.circleci/config.yml

```diff
@@ -0,0 +1,28 @@
+# Python CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  test:
+    docker:
+      - image: mbemis/bond:python2.7_gcloud
```

Comment:
had to take the circle python 2.7 image and then shove the google cloud sdk and google app engine and the "original local development server" into it.

i know this isn't the permanent place for this to live, but for the moment it works.
base file:

my additions: https://github.com/CircleCI-Public/circleci-dockerfiles/blob/master/python/images/2.7.15/Dockerfile

```
# install gcloud sdk
RUN curl https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz > /tmp/google-cloud-sdk.tar.gz
RUN mkdir -p /usr/local/gcloud
RUN tar -C /usr/local/gcloud -xvf /tmp/google-cloud-sdk.tar.gz
RUN /usr/local/gcloud/google-cloud-sdk/install.sh --quiet
ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin

#install original local development server
RUN curl https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.70.zip > tmp/google-appengine-sdk.zip
RUN mkdir -p /usr/local/google_appengine
RUN unzip tmp/google-appengine-sdk.zip -d /usr/local
ENV PATH $PATH:/usr/local/google_appengine
```